apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -1124,6 +1124,18 @@ events:
     osver: {ios: {min: "9.0"}}
     since: '5.1.0'
 
+  - name: handleurl
```

Comment:
@cheekiatng In case you ask why we need this here: It is triggered by https://jira.appcelerator.org/browse/TIMOB-23884 that is caused by an open Apple issue. It is both workaround until Apple fixes it as well as the future-way to receive updates from URL's that get handled by the application.